### PR TITLE
prepare for version 0.4.1

### PR DESCRIPTION
**Pull request by YJDoc2**
This uptick is primarily due to https://github.com/containers/youki/issues/2894 , our nc update in last release broke musl builds, which was them fixed by updating to nc's latest patch releases which fix the issues. As this is just a dep update and no breaking changes, only need to change minor version.

### CODE REVIEW

**Comment by YJDoc2** [comment]
Waiting for reply to https://github.com/containers/youki/issues/2894#issuecomment-2311567071 before merging this.

**Review by utam0k — APPROVED** [review]

**Pull request merged**
